Makefile

```diff
@@ -18,13 +18,16 @@ ifeq (,$(filter %$(BUILD_META),$(TAG)))
 $(error TAG needs to end with build metadata: $(BUILD_META))
 endif
 
+GOLANG_VERSION := $(shell if echo $(TAG) | grep -qE '^v1\.(18|19|20)\.'; then echo v1.15.15b5; else echo v1.16.7b7; fi)
```

Comment:
```suggestion
GOBORING_VERSION := $(shell if echo $(TAG) | grep -qE '^v1\.(18|19|20)\.'; then echo v1.15.15b5; else echo v1.16.7b7; fi)
```

Reply:
That might have been a better choice, but at this point I'm matching the pattern from
https://github.com/rancher/image-build-kubernetes/blob/cbba8640bad5c678ae88d660aecabbea4aa94d10/Makefile#L19